build.gradle

```diff
@@ -0,0 +1,176 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3"
+  }
+}
+
+plugins {
+  id 'checkstyle'
+  id 'io.spring.dependency-management' version '1.0.5.RELEASE'
+  id 'org.springframework.boot' version '2.0.6.RELEASE'
+  id 'com.github.ben-manes.versions' version '0.20.0'
+}
+
+apply plugin: 'checkstyle'
+apply plugin: 'java-library'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'maven'
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+group 'uk.gov.hmcts.reform'
+version '5.0.0'
+
+dependencyUpdates.resolutionStrategy = {
+  componentSelection { rules ->
+    rules.all { ComponentSelection selection ->
+      boolean rejected = ['alpha', 'beta', 'rc', 'cr', 'm'].any { qualifier ->
+        selection.candidate.version ==~ /(?i).*[.-]${qualifier}[.\d-]*/
+      }
+      if (rejected) {
+        selection.reject('Release candidate')
+      }
+    }
+  }
+}
+
+checkstyle {
+  maxWarnings = 0
+  toolVersion = '8.3'
```

Comment:
Please update to latest version, I believe 8.19 is out now.

Reply:
👍